contracts/test/poolBooster/merkl-pool-booster.sonic.fork-test.js

```diff
@@ -27,7 +27,7 @@ describe("ForkTest: Merkl Pool Booster", function () {
     oSonicVault = fixture.oSonicVault;
     oSonic = fixture.oSonic;
     poolBoosterMerklFactory = fixture.poolBoosterMerklFactory;
-    governor = fixture.governor;
+    governor = await impersonateAndFund(addresses.multichainStrategist);
```

Comment:
would be nice to rename the variable to `strategist` or `guardian` to avoid confusion in future

Reply:
[424a4a4](https://github.com/OriginProtocol/origin-dollar/pull/2611/commits/424a4a45665a525eacad838a54cb7f5b75c8f8ba)